src/defaultOperations.ts

```diff
@@ -0,0 +1,62 @@
+import axios from "axios";
+class DefaultOperations<T> {
+  BASE_URL: String = "https://api.statuspage.io/v1/pages";
+  path: String;
+  pageId: String;
+  config: Object;
+
+  constructor(pageId: String, apiKey: String, path: String) {
+    this.path = path;
+    this.pageId = pageId;
+    this.config = {
+      headers: {
+        Authorization: `OAuth ${apiKey}`,
+      },
+    };
+  }
+
+  async get(id: String): Promise<T> {
+    let res = await axios.get(
```

Comment:
I think you should use 'const' instead of 'let' whenever the variable is not reassigned.

Reply:
There are some other small things that should appear as a problem with eslint (trailing commas, for example).